pkg/models/task_search.go

```diff
@@ -44,6 +99,20 @@ type dbTaskSearcher struct {
 	hasFavoritesProject bool
 }
 
+func (sf *SubTableFilter) ToBaseSubQuery() *builder.Builder {
+	var cond = builder.
+		Select("1").
+		From(sf.Table).
+		Where(builder.And(builder.Expr(sf.BaseFilter)))
```

Comment:
No need for `builder.And` if there's always only one condition.

Reply:
Fixed